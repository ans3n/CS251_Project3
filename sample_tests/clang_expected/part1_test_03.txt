insert xn3ZpHY krfKnkt
peek xn3ZpHY
krfKnkt
size
1
peek xn3ZpHY
krfKnkt
bucket_count
15
insert XgO6U8g 63Fcpz
insert 8m0e6 GNybbzT
insert geuqYFr4 6SPpzr2P
print
  0: [empty]
  1: (1) xn3ZpHY -> krfKnkt
  2: [empty]
  3: (3) 8m0e6 -> GNybbzT
  4: [empty]
  5: (5) geuqYFr4 -> 6SPpzr2P
  6: [empty]
  7: [empty]
  8: [empty]
  9: [empty]
 10: [empty]
 11: (11) XgO6U8g -> 63Fcpz
 12: [empty]
 13: [empty]
 14: [empty]
insert gp9XU5b EGgwgbT1
extract xn3ZpHY
krfKnkt
size
4
resize 4
extract XgO6U8g
63Fcpz
hash_code 8m0e6
1
extract gp9XU5b
EGgwgbT1
print
  0: [empty]
  1: (1) 8m0e6 -> GNybbzT
  2: (2) geuqYFr4 -> 6SPpzr2P
  3: [empty]
empty
false
insert CYXtB9Jf qMmc
insert LPPBeXu qq2k
hash_code LPPBeXu
2
print
  0: (2) LPPBeXu -> qq2k
  1: (1) 8m0e6 -> GNybbzT
  2: (2) geuqYFr4 -> 6SPpzr2P
  3: (1) CYXtB9Jf -> qMmc
bucket_count
4
peek CYXtB9Jf
qMmc
insert pf5gbJ LEPt8xr
print
  0: [empty]
  1: [empty]
  2: (2) LPPBeXu -> qq2k
  3: [empty]
  4: (4) pf5gbJ -> LEPt8xr
  5: (5) 8m0e6 -> GNybbzT
  6: (6) geuqYFr4 -> 6SPpzr2P
  7: (5) CYXtB9Jf -> qMmc
extract pf5gbJ
LEPt8xr
insert 0KqfQc1F AEmLgK
insert ySc7j HAV3jNER
peek 8m0e6
GNybbzT
insert 60S9rh kUct
size
7
quit
