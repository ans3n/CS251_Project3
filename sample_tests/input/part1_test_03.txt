string string
15
insert xn3ZpHY krfKnkt
peek xn3ZpHY
size
peek xn3ZpHY
bucket_count
insert XgO6U8g 63Fcpz
insert 8m0e6 GNybbzT
insert geuqYFr4 6SPpzr2P
print
insert gp9XU5b EGgwgbT1
extract xn3ZpHY
size
resize 4
extract XgO6U8g
hash_code 8m0e6
extract gp9XU5b
print
empty
insert CYXtB9Jf qMmc
insert LPPBeXu qq2k
hash_code LPPBeXu
print
bucket_count
peek CYXtB9Jf
insert pf5gbJ LEPt8xr
print
extract pf5gbJ
insert 0KqfQc1F AEmLgK
insert ySc7j HAV3jNER
peek 8m0e6
insert 60S9rh kUct
size
quit
