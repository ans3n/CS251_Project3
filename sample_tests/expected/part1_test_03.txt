insert xn3ZpHY krfKnkt
peek xn3ZpHY
krfKnkt
size
1
peek xn3ZpHY
krfKnkt
bucket_count
15
insert XgO6U8g 63Fcpz
insert 8m0e6 GNybbzT
insert geuqYFr4 6SPpzr2P
print
  0: [empty]
  1: [empty]
  2: [empty]
  3: [empty]
  4: [empty]
  5: [empty]
  6: (6) geuqYFr4 -> 6SPpzr2P
  7: (7) 8m0e6 -> GNybbzT
  8: [empty]
  9: (9) xn3ZpHY -> krfKnkt
 10: [empty]
 11: [empty]
 12: (12) XgO6U8g -> 63Fcpz
 13: [empty]
 14: [empty]
insert gp9XU5b EGgwgbT1
extract xn3ZpHY
krfKnkt
size
4
resize 4
extract XgO6U8g
63Fcpz
hash_code 8m0e6
2
extract gp9XU5b
EGgwgbT1
print
  0: (0) geuqYFr4 -> 6SPpzr2P
  1: [empty]
  2: (2) 8m0e6 -> GNybbzT
  3: [empty]
empty
false
insert CYXtB9Jf qMmc
insert LPPBeXu qq2k
hash_code LPPBeXu
0
print
  0: (0) geuqYFr4 -> 6SPpzr2P
  1: (0) CYXtB9Jf -> qMmc
  2: (2) 8m0e6 -> GNybbzT
  3: (0) LPPBeXu -> qq2k
bucket_count
4
peek CYXtB9Jf
qMmc
insert pf5gbJ LEPt8xr
print
  0: (0) geuqYFr4 -> 6SPpzr2P
  1: (0) CYXtB9Jf -> qMmc
  2: (2) pf5gbJ -> LEPt8xr
  3: [empty]
  4: (4) LPPBeXu -> qq2k
  5: [empty]
  6: (6) 8m0e6 -> GNybbzT
  7: [empty]
extract pf5gbJ
LEPt8xr
insert 0KqfQc1F AEmLgK
insert ySc7j HAV3jNER
peek 8m0e6
GNybbzT
insert 60S9rh kUct
size
7
quit
